src/locales/ja.yaml

```diff
@@ -16,9 +16,9 @@ app:
       not_loaded: メッシュを読込しない
     tooltip:
       calibrate: 新しいキャリブレーションを開始し、プロファイル'default'として保存します
-      delete: プロファイルを削除します
+      delete: プロファイルを削除します。これにより、プリンタが再起動します
```

Comment:
The original in the en.yaml is "Delete Profile", this doesn't seem correct given that.

Reply:
I will fix it tomorrow. please wait a moment